src/streaming/transcoders.ts

```diff
@@ -0,0 +1,322 @@
+import { sendDelete, sendGet, sendPost, sendPut } from './internal/httpHelpers';
+import * as Urls from '../urls';
+import {
+    ListTranscodersSortOptions,
+    ListTranscodersInstancesSortOptions,
+    ListTranscodersProfilesSortOptions,
+    Transcoder,
+    TranscoderProfile,
+    TranscoderInstance,
+    CreateTranscoderRequest,
+    UpdateTranscoderRequest,
+} from './types/transcoders';
+
+/**
+ * Lists all transcoders.
+ *
+ * @link https://docs.dolby.io/streaming-apis/reference/transcoder_listtranscoders
+ * @beta
+ * @remarks Cloud transcoder usage is not currently available for general usage.
+ * If you would like to opt in, please contact our Sales team.
+ *
+ * @param apiSecret The API Secret used to authenticate this request.
+ * @param options Options to sort the response.
+ *
+ * @returns A {@link !Promise Promise} whose fulfillment handler receives an array of {@link Transcoder} objects.
+ */
+export const listTranscoders = async (apiSecret: string, options: ListTranscodersSortOptions): Promise<Transcoder[]> => {
+    const params = {
+        sortBy: options.sortBy,
+        page: options.page.toString(),
+        itemsOnPage: options.itemsOnPage.toString(),
+        isDescending: (options.isDescending ?? false).toString(),
+    };
+
+    if (options.status) {
+        params['status'] = options.status;
+    }
+
+    const queryOptions = {
+        hostname: Urls.getRtsHostname(),
+        path: '/api/transcoders',
```

Comment:
Noting that this is correct, it isn't like some of the other endpoints that are of the rpc style form `/api/resource/list`